python/ambassador/compile.py

```diff
@@ -31,6 +31,10 @@ class _CompileResult(TypedDict):
     xds: NotRequired[EnvoyConfig]
 
 
+def null_file_checker(path: str) -> bool:
```

Comment:
```suggestion
def null_file_checker(_: str) -> bool:
```

Tiny nit - unused variables shouldn't be bound to names